pkg/agent/const.go

```diff
@@ -86,6 +86,8 @@ const (
 	kubernetesKubeletMonitorSystemdService = "linux/cloud-init/artifacts/kubelet-monitor.service"
 	kubernetesDockerMonitorSystemdTimer    = "linux/cloud-init/artifacts/docker-monitor.timer"
 	kubernetesDockerMonitorSystemdService  = "linux/cloud-init/artifacts/docker-monitor.service"
+	labelsScript                           = "linux/cloud-init/artifacts/labels.sh"      // Note(charliedmcb): I have not completely followed this path. There might be more that has to be added
```

Comment:
remove note after testing...

Reply:
I believe I actually tracked down how this was used awhile ago, and changed how it was handled.

I'll remove it when I'm ready to merge :)

Reply:
Removed